packages/wonder-blocks-forms/components/dropdown.js

```diff
@@ -0,0 +1,121 @@
+// @flow
```

Comment:
The behavior of this component on the whole is really hacky, please focus on its ability to switch between being controlled vs uncontrolled as opposed to the user facing behavior of the component itself.